README.md

```diff
@@ -1,16 +1,44 @@
-# Title
+# Advanced Synchrony Overview
 
-## Learning Goals
+## Looking Back
 
--SWBAT 1
--SWBAT 2
+Earlier in our front-end web programming journey, we learned about the
+asynchronous model of program execution. In this model, functions can perform
+tasks while other tasks are completing in the background. This creates the basis
+for the efficient experience JavaScript provides for web users—without it, we
+would be waiting for animated gifs to load or audio to continue to stream while
+browsers finished responding to an earlier request.
 
-## Introduction
+To implement an asynchronous model in JavaScript, we learned how to use the
+Fetch API. The process of using `fetch()` gives us these steps:
 
-## SWBAT 1
+- Make a request
+- Handle the data sent back
+- Handle any errors the server sends
 
-## SWBAT 2
+In code, it looks something like this:
 
-## Conclusion
+```javascript
+fetch('http://example.com/log_dogger', configObj)
+	.then(function(response) {
+		return response.text();
+	})
+	.then(function(content) {
+		if (response == 'OK') {
+			console.log('Added a new dog!');
+		}
+	})
+	.catch(function(error) {
+		alert('Bad things! Ragnarők!');
+		return 'This is an error message. There are many like it, but this one is for log_dogger';
+	});
+```
 
```

Comment:
This code snippet errors due to `configObj`. It might be better to just go with fetch('http://example.com/log_dogger') which will get caught with the 'Bad things! Ragnarok!' alert if a student were to copy and paste this code into a console.

This might also be a good place to connect the Intermediate JS prerequisite - something like... 'Applying what we learned about arrow functions, we can now write this as:'